src/releasing/packager.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# Source this first as it has the potential to scribble all over our variables otherwise
+source /etc/os-release || exit 1
+
+function echo_warning {
+  echo -e "\033[1;33mWARNING\033[0m  $1"
+}
+
+function echo_error {
+  echo -e "\033[1;31mERROR\033[0m  $1"
+}
+
+function echo_success {
+  echo -e "\033[1;32mSUCCESS\033[0m  $1"
+}
+
+function echo_info {
+  echo -e "\033[1;34mINFO\033[0m  $1"
+}
+
+function exit_usage {
+    echo
+    echo "USAGE: releaser.sh RELEASE_NUMBER [RELEASE_CANDIDATE]"
+    echo "       RELEASE_NUMBER should be of the form YY.MM.N without leading zeros"
+    exit 3
+}
+
+if [[ ! $ID_LIKE =~ 'rhel' ]]; then
+    echo_error "We do not currently support packing on an non-RHEL like OS (this OS claims to be like '$ID_LIKE')."
+    exit 1
+fi
+
+PACKAGE_SUFFIX="el$VERSION_ID"
+
+echo_info "Set package suffix to $PACKAGE_SUFFIX"
+
+
+# Now define our environment
+REPOS_MVN="release aii CAF CCM cdp-listend configuration-modules-core configuration-modules-grid LC ncm-cdispd ncm-ncd ncm-query ncm-lib-blockdevices"
```

Comment:
normally, i'd ask to factor the common bits. not sure how easy it is to do?

Reply:
I'll look at doing this in a different PR.